sdcm/utils/adaptive_timeouts/load_info_store.py

```diff
@@ -182,7 +181,7 @@ def __init__(self):
     def store(self, metrics: dict[str, Any], operation: str, duration: float, timeout: float,
               timeout_occurred: bool):
         body = metrics
-        body["test_id"] = self._test_id
```

Comment:
like this:?
`body["timestamp"] = time.time()`

Reply:
more like
```python
datetime.utcfromtimestamp(time.time())
```

FYI see the nemesis_publisher.py code, as for the data it's saves, I think it might be useful in your case as well, for looking at the data, and filtering specific tests, or specific scylla versions.

```
        new_nemesis_data = dict(
            test_id=test_data['test_details']['test_id'],
            job_name=test_data['test_details']['job_name'],
            test_name=test_data['test_details']['test_name'],
            scylla_version=version,
            scylla_git_sha=commit_id,
        )
```

Reply:
done